package.json

```diff
@@ -36,6 +36,7 @@
         "@react-native/metro-config": "0.73.5",
         "@react-native/typescript-config": "0.73.1",
         "@testing-library/react-native": "7.0.2",
+        "@types/jest": "^29.5.12",
```

Comment:
Was this a transitive dependency before?

Reply:
I believe it was just missing, making the IDE clueless about the correctness of the test files, so I simply added it in the dev dependencies.